.github/workflows/docker_update_base_windows.yml

```diff
@@ -7,11 +7,11 @@ on:
   workflow_dispatch:
     inputs:
       antrea-repository:
-        description: 'The Antrea repository to check-out'
+        description: 'The Antrea repository to check-out, a forked Antrea repo should be like "$username/antrea"'
```

Comment:
```suggestion
        description: 'The Antrea repository to check-out; it will typically be a personal Antrea fork such as '<your-github-user>/antrea'
```

Reply:
Updated.